src/os/filestore/LFNIndex.cc

```diff
@@ -472,12 +483,23 @@ int LFNIndex::list_subdirs(const vector<string> &to_list,
 			   vector<string> *out)
 {
   string to_list_path = get_full_path_subdir(to_list);
+  int r = 0;
```

Comment:
nit, might want to move the definition of `r` closer to where it is used. i.e., to move to right before the `while()` loop.

Reply:
thx, I will update code

Reply:
pushed.